src/dftbp/dftbplus/main.F90

```diff
@@ -4822,9 +4818,7 @@ subroutine getNextInputDensityCplx(env, ints, neighbourList, nNeighbourSK, dense
         #:endif
 
           ! Construct sparse density matrix for later Mulliken analysis
-          call getSparseSize(neighbourList%iNeighbour, nNeighbourSK, img2CentCell, orb,&
-              & sparseSize)
```

Comment:
If we remove this call, we don't need to import it anymore and should probably even remove its definition as this is the only instance `getSparseSize` is invoked.

Reply:
@vanderhe OK, function removed and rebase to current main.

Reply:
thx